### ISSUE TITLE
Android: F-Droid release

### ISSUE THREAD

**Issue by HiWhatName** [yuzu-emu/yuzu#10535]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What feature are you suggesting?

# Possible [F-Droid](https://f-droid.org/) release?

Considering there're tons of devices out there, which don't support the Google Playstore, will we see an release of the Yuzu emulator for an android on [F-Droid](https://f-droid.org/)?

Having to use proprietary to software to keep an **FOSS** tool up-to date seems incoherent

### Why would this feature be useful?

It'll enable users with custom roms and those, who're unable to use the Google Playstore for whatever reason to use **Yuzu**.

**Comment by HiWhatName** [comment]
>  but is it not possible for you to use the Aurora Store to install/update yuzu Android releases at least for the time being?

**Possible:** Probally?
**Recommended:** Not at all, all aurora Store anonymous accounts got hit with a ban hammer[ a few days ago.](https://www.reddit.com/r/degoogle/comments/13td3iq/)

**TLDR;** You'd be risking your google account

**Comment by ClockGen** [comment]
Hello, I checked yuzu source code for compliance with F-Droid inclusion criteria (https://f-droid.org/en/docs/Inclusion_Policy/) and yuzu seems to fit fine. I also wrote build instructions and tested if yuzu builds on f-droid buildserver, so if the developers want yuzu in F-Droid, it's certainly possible.

**Comment by t895** [comment]
I'm waiting on other approval but I'd love this to happen. And for getting the right version information, just set an `autoVersioned` property in `local.properties`

Look here in the app level build.gradle
![Screenshot_20231008-140502.png](https://github.com/yuzu-emu/yuzu/assets/14132249/e57a8541-0faa-4d43-8159-b29cf34f2aab)

**Comment by ClockGen** [comment]
> I'm waiting on other approval but I'd love this to happen. And for getting the right version information, just set an `autoVersioned` property in `local.properties`
> 
> Look here in the app level build.gradle ![Screenshot_20231008-140502.png](https://user-images.githubusercontent.com/14132249/273466030-e57a8541-0faa-4d43-8159-b29cf34f2aab.png)

Thanks, I already do that in metadata.

**Comment by t895** [comment]
Oh sorry I misunderstood your comment

**Comment by ClockGen** [comment]
Well, right now everything seems to work, but as per F-Droid policy I need project developers' permission to continue.

**Comment by nicolasmaia** [comment]
@liamwhite Is it OK to go ahead with it?

**Comment by liamwhite** [comment]
@nicolasmaia I am not the right person to ask - that would be @bunnei

**Comment by HiWhatName** [comment]
Hey @flamesage, mind taking a look at the changes (**F-Droid metadata**) made by @ClockGen?
Given that @bunnei _hasn't been active_ on any public repos since _August_ & you seem to be part of the team.

Except for the failing _linting_ test, everything **lgtm** :)

**Comment by FearlessTobi** [comment]
Flame Sage has not been active in yuzu or Citra for several years.

**Comment by guoyunhe** [comment]
Hi, Google Play is blocked in China. Here is no easy way to use Google Play or Aurora Store. Using a VPN is illegal and doesn't always work today.

Is it possible to provide direct APK file download from the website? Or publish it on F-Droid?

**Comment by FearlessTobi** [comment]
@guoyunhe Here you go :)

https://github.com/yuzu-emu/yuzu-android/releases

**Comment by oelmekki** [comment]
@FearlessTobi as someone who is using LineageOs without the play store, I thank you heartedly as well. :)

**Comment by ClockGen** [comment]
Any update on this? I didn't work on metadata yet since I'm waiting for a maintainer's permission.

**Comment by t895** [comment]
@ClockGen  Discussed this internally (really sorry about the delay). Our preference is that if we were to publish the app to F-Droid, we would do it ourselves officially. We're currently working through some other changes right now, but will consider an F-Droid update in the coming months.
We'll leave this issue open for now so that it stays on our radar.

**Comment by guoyunhe** [comment]
@t895 F-Droid supports [creating custom repository](https://f-droid.org/tutorials/create-repo/). Users can add your repository URL to the F-Droid app. A bit manual work for users, but easier for developers. You can setup a fully automatic CI to do the whole publishing process.

**Comment by ClockGen** [comment]
> @ClockGen Discussed this internally (really sorry about the delay). Our preference is that if we were to publish the app to F-Droid, we would do it ourselves officially. We're currently working through some other changes right now, but will consider an F-Droid update in the coming months. We'll leave this issue open for now so that it stays on our radar.

Ok, if you have any questions about F-Droid packaging process, feel free to ask.

**Comment by inson1** [comment]
I hope you are going to do also reporoducible builds
